examples/qm9/predict_qm9.py

```diff
@@ -155,10 +159,13 @@ def postprocess_fn(x):
     # --- evaluate ---
     # To calc loss/accuracy, we can use `Evaluator`, `ROCAUCEvaluator`
     print('Evaluating...')
-    val_iterator = SerialIterator(val, 16, repeat=False, shuffle=False)
+    test_iterator = SerialIterator(test, 16, repeat=False, shuffle=False)
     eval_result = Evaluator(
-        val_iterator, regressor, converter=concat_mols, device=args.gpu)()
+        test_iterator, regressor, converter=concat_mols, device=args.gpu,
+        name='test', eval_func=regressor.predictor)()
```

Comment:
`name` is not in Evaluator.
https://github.com/chainer/chainer/blob/master/chainer/training/extensions/evaluator.py#L79